asv_bench/benchmarks/pandas/arithmetic.py

```diff
@@ -0,0 +1,379 @@
+import operator
+import warnings
+
+import numpy as np
+
+import modin.pandas as pd
+from modin.pandas import (
+    DataFrame,
+    Series,
+    date_range,
+)
+
+from .pandas_vb_common import numeric_dtypes
+
+import pandas.tseries
+from pandas.tseries import offsets
+try:
+    import pandas.tseries.holiday
+except ImportError:
+    pass
+
+
+class IntFrameWithScalar:
+    params = [
+        [np.float64, np.int64],
+        [2, 3.0, np.int32(4), np.float64(5)],
+        [
+            operator.add,
+            operator.sub,
+            operator.mul,
+            operator.truediv,
+            operator.floordiv,
+            operator.pow,
+            operator.mod,
+            operator.eq,
+            operator.ne,
+            operator.gt,
+            operator.ge,
+            operator.lt,
+            operator.le,
+        ],
+    ]
+    param_names = ["dtype", "scalar", "op"]
+
+    def setup(self, dtype, scalar, op):
+        arr = np.random.randn(20000, 100)
+        self.df = DataFrame(arr.astype(dtype))
+
+    def time_frame_op_with_scalar(self, dtype, scalar, op):
+        op(self.df, scalar)
+
+
+class OpWithFillValue:
+    def setup(self):
+        # GH#31300
+        arr = np.arange(10**6)
+        df = DataFrame({"A": arr})
+        ser = df["A"]
+
+        self.df = df
+        self.ser = ser
+
+    def time_frame_op_with_fill_value_no_nas(self):
+        self.df.add(self.df, fill_value=4)
+
+    def time_series_op_with_fill_value_no_nas(self):
+        self.ser.add(self.ser, fill_value=4)
+
+
+class MixedFrameWithSeriesAxis:
+    params = [
+        [
+            "eq",
+            "ne",
+            "lt",
+            "le",
+            "ge",
+            "gt",
+            "add",
+            "sub",
+            "truediv",
+            "floordiv",
+            "mul",
+            "pow",
```

Comment:
```suggestion
            "add",
            "mul",
```
I think we can leave 1 or 2 operations because they go through the same code path.